### PR TITLE
Identify layer changes

### PR DESCRIPTION
**Pull request by singhgag**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Edit data/Delete features (feature service)/DeleteFeaturesViewController.swift [hunk 1]
```diff
@@ -96,28 +96,17 @@ class DeleteFeaturesViewController: UIViewController, AGSMapViewTouchDelegate, A
...
-        let queryParams = AGSQueryParameters()
-        queryParams.geometry = env
-        
-        self.lastQuery = self.featureTable.queryFeaturesWithParameters(queryParams, completion: { [weak self] (result:AGSFeatureQueryResult?, error:NSError?) -> Void in
+        self.lastQuery = self.mapView.identifyLayer(self.featureLayer, screenCoordinate: screen, tolerance: 44, maximumElements: 1) { [weak self] (geoElement: [AGSGeoElement]?, error: NSError?) -> Void in
             if let error = error {
                 print(error)
             }
-            else if let feature = result?.nextObject() {
+            else if let features = geoElement as? [AGSFeature] where features.count > 0 {
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Edit data/Delete features (feature service)/DeleteFeaturesViewController.swift]: This should be geoElements (plural)

File: arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift [hunk 2]
```diff
@@ -78,24 +78,17 @@ class EditFeaturesOnlineViewController: UIViewController, AGSMapViewTouchDelegat
...
-        let tolerance:Double = 22
-        let mapTolerance = tolerance * self.mapView.unitsPerPixel
-        let envelope = AGSEnvelope(XMin: mappoint.x - mapTolerance, yMin: mappoint.y - mapTolerance, xMax: mappoint.x + mapTolerance, yMax: mappoint.y + mapTolerance, spatialReference: self.map.spatialReference)
-        let queryParams = AGSQueryParameters()
-        queryParams.geometry = envelope
-        
 
-        self.lastQuery = self.featureLayer.featureTable!.queryFeaturesWithParameters(queryParams){ [weak self] (queryResult, error) in
+        self.lastQuery = self.mapView.identifyLayer(self.featureLayer, screenCoordinate: screen, tolerance: 44, maximumElements: 10) { [weak self] (geoElements: [AGSGeoElement]?, error: NSError?) -> Void in
             if let error = error {
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift]: Redundant error handling. There is an `else if let error = error` below

File: arcgis-ios-sdk-samples/Edit data/Edit feature attachments/EditAttachmentViewController.swift [hunk 3]
```diff
@@ -61,22 +61,12 @@ class EditAttachmentViewController: UIViewController, AGSMapViewTouchDelegate, A
...
-        let queryParams = AGSQueryParameters()
-        queryParams.geometry = envelope
-        
-        self.lastQuery = self.featureTable.queryFeaturesWithParameters(queryParams, completion: { [weak self] (result:AGSFeatureQueryResult?, error:NSError?) -> Void in
+        self.lastQuery = self.mapView.identifyLayer(self.featureLayer, screenCoordinate: screen, tolerance: 44, maximumElements: 1) { [weak self] (geoElement: [AGSGeoElement]?, error: NSError?) -> Void in
             if let error = error {
                 print(error)
             }
-            else if let feature = result?.nextObject() as? AGSArcGISFeature {
+            else if let features = geoElement as? [AGSArcGISFeature] where features.count > 0 {
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Edit data/Edit feature attachments/EditAttachmentViewController.swift]: should be geoElements

**Comment by dg0yal** [comment]
Rest all looks good. Nice work.

File: arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift (hunk 2 above)
Comment (singhgag) [inline arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift]: @dg0yal I didn't get you

Comment (dg0yal) [inline arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift]: You have another error check below on line 109. Do we need both?

Comment (singhgag) [inline arcgis-ios-sdk-samples/Edit data/Edit Features (connected)/EditFeaturesOnlineViewController.swift]: We don't. I will remove it. Thanks.

**Pull request merged**